### PR TITLE
{cae}[system/system] ABAQUS v2021 (hotfix 2132)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires:
* https://github.com/easybuilders/easybuild-easyblocks/pull/2562
* https://github.com/easybuilders/easybuild-easyblocks/pull/2569

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3528.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e8369eafaa7932f5ceeb1d4d5a0a2bc6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2605.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/b286101d737df6e30a7ac001e561dcbb for a full test report.

File: easybuild/easyconfigs/a/ABAQUS/ABAQUS-2021-hotfix-2132.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name = 'ABAQUS'
+version = '2021'
+local_hotfix = '2132'
+versionsuffix = '-hotfix-%s' % local_hotfix
+
+homepage = 'http://www.simulia.com/products/abaqus_fea.html'
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/a/ABAQUS/ABAQUS-2021-hotfix-2132.eb:6]:
```suggestion
homepage = 'https://www.simulia.com/products/abaqus_fea.html'
```

**Review by SebastianAchilles — COMMENTED** [review]
I think we can use `https` here.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**